po/clubhouse.pot

```diff
@@ -0,0 +1,175 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the clubhouse package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: clubhouse\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-05 13:25+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/achievement-summary-view.ui:33
+msgid "Achievement Badge Name"
+msgstr ""
+
+#: data/achievement-summary-view.ui:49
+msgid "Description"
+msgstr ""
+
+#: data/achievements-view.ui:66
+msgid "Achievements"
+msgstr ""
+
+#: data/achievements-view.ui:131
+msgid "page0"
+msgstr ""
+
+#: data/achievements-view.ui:163
+msgid "page1"
+msgstr ""
+
+#: data/activity-card.ui:219
+msgid "play"
+msgstr ""
+
+#: data/character-view.ui:81 data/clubhouse-window.ui:255
+#: data/clubhouse-window.ui:306
+msgid "Clubhouse"
+msgstr ""
+
+#: data/character-view.ui:124
+msgid "Workspace"
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:102
+msgid "Hack extension is not enabled. Click here to solve"
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:115
+msgid ""
+"The Hack GNOME Shell Extension is not installed. <b>Install extension</b>."
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:133
+msgid "Installing hack extension…"
+msgstr ""
+
+#: data/clubhouse-window.ui:41 eosclubhouse/clubhouse.py:108
+msgid "Games"
+msgstr ""
+
+#: data/clubhouse-window.ui:87 eosclubhouse/clubhouse.py:102
+msgid "Art"
+msgstr ""
+
+#: data/clubhouse-window.ui:133 eosclubhouse/clubhouse.py:126
+msgid "Web"
+msgstr ""
+
+#: data/clubhouse-window.ui:179 eosclubhouse/clubhouse.py:120
+msgid "Maker"
+msgstr ""
+
+#: data/clubhouse-window.ui:225
+msgid "Operating System"
+msgstr ""
+
+#: data/clubhouse-window.ui:342
+msgid "Pathways"
+msgstr ""
+
+#: data/clubhouse-window.ui:369
+msgid "Hack News"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:4
+#: data/com.hack_computer.Clubhouse.metainfo.xml:7
+msgid "Hack"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:5
+msgid "eos-clubhouse"
```

Comment:
This looks like an internal name. Better change the desktop file to be "GenericName=Clubhouse" or "Hack".